node/silkworm/common/log.cpp

```diff
@@ -30,6 +30,7 @@ namespace silkworm::log {
 static Settings settings_{};
 static std::mutex out_mtx{};
 static std::unique_ptr<std::fstream> file_{nullptr};
+thread_local std::string thread_name_{};
```

Comment:
Why not declare this inside the `get_thread_name` function?

Reply:
Cause it can be valued also by `set_thread_name`